CHANGELOG.md

```diff
@@ -1465,7 +1466,7 @@ The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-co
 | Honeycomb | Kinesis |
 | Lightstep |
 | New Relic |
-| SAPM (SignalFx APM) | 
```

Comment:
While I appreciate those changes, they are just too numerous and shouldn't be included in this PR.

Reply:
Okay, I have reverted the changes.